R/pkg/R/generics.R

```diff
@@ -623,6 +623,10 @@ setGeneric("getItem", function(x, ...) { standardGeneric("getItem") })
 
 #' @rdname column
 #' @export
+setGeneric("isNaN", function(x) { standardGeneric("isNaN") })
```

Comment:
open https://issues.apache.org/jira/browse/SPARK-12118 to explain in release note.